radio/src/targets/common/arm/stm32/usbd_hid_joystick.c

```diff
@@ -110,7 +110,7 @@ static const uint8_t  *USBD_HID_GetCfgDesc (uint8_t speed, uint16_t *length);
 __ALIGN_BEGIN static const uint8_t HID_JOYSTICK_ReportDesc[] __ALIGN_END =
 {
     0x05, 0x01,                    //     USAGE_PAGE (Generic Desktop)
-    0x09, 0x05,                    //     USAGE (Game Pad)
+    0x09, 0x04,                    //     USAGE (Joystick)
```

Comment:
It is 0x05 in EdgeTX, I'm not against the change but is there a specific reason to change it?

Reply:
Value from original firmware, IMHO more suitable for the use. Doesn't do much however.

Reply:
As this is port of OpenTX/EdgeTX and there is no specific reason to change that, it  should stay as it was.